package/harvester-os/files/etc/systemd/system/systemd-time-wait-sync.service.d/override.conf

```diff
@@ -0,0 +1,2 @@
+[Service]
+TimeoutStartSec=180
```

Comment:
Maybe we should make it be `oneshot` service to prevent the service restart dependency?
Also, could we add `Description ` in the [Unit] section to clarify this service? Thanks!

Reply:
We should not modify systemd service to `oneshot` to make sure the behavior.
and `/lib/systemd/system/systemd-time-wait-sync.service` has original `Description` that we should not override it.
I will consider to keep those as original systemd services as possible.
and users could check the systemd docs to know the behavior

Reply:
I just found `systemd-time-wait-sync.service` is already an oneshot service.

```
[Service]
Type=oneshot
ExecStart=/lib/systemd/systemd-time-wait-sync
TimeoutStartSec=infinity
RemainAfterExit=yes
```

Reply:
Ah... I missing something. That is already a oneshot service. That's no question here.

Reply:
Thanks for this detail!

Reply:
The dependency issues could happen because serval services depend on each other. Once you restart someone, another will also be restarted.

Reply:
> Once you restart someone, another will also be restarted
IIRC, systemd didn't do that until service has "PartOf" attribute.

Reply:
and oneshot service only said it is an one time command which is not a continuous process.
It's almost the same as "simple" service.

Reply:
And I used "Wants" to make sure the time sync is better to have but not requirement.
So after timeout, time sync service will be mark as failed.
and rke2-server/agent will ignore that service.